src/cargo/util/hasher.rs

```diff
@@ -1,25 +1,3 @@
-//! Implementation of a hasher that produces the same values across releases.
-//!
-//! The hasher should be fast and have a low chance of collisions (but is not
-//! sufficient for cryptographic purposes).
```

Comment:
Should we keep this part of the comment?

Reply:
Added it back!